projects/packages/password-checker/src/class-jetpack-password-checker.php

```diff
@@ -0,0 +1,1290 @@
+<?php
+/**
+ * Jetpack Password Checker package.
+ *
+ * @package  automattic/jetpack-password-checker
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Checks passwords strength.
+ */
+class Jetpack_Password_Checker {
```

Comment:
This should be renamed to Password_Checker

as well as the file name class-password-checker.php

Reply:
Done.